tests/Support/MailsMockTrait.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace RonasIT\TelescopeExtension\Tests\Support;
+
+use Illuminate\Mail\Mailable;
+
+trait MailsMockTrait
+{
+    protected function assertFixture(array $expectedMailData, Mailable $mail, bool $exportMode = false): void
```

Comment:
let's proceed with https://github.com/RonasIT/laravel-helpers/issues/228 at first, seems LaravelHelpers cpakge's fix will help to avoid to create this trait